### PR TITLE
Feature: accept asyncValidation property

### PR DESCRIPTION
**Pull request by jvhoven**

### CODE REVIEW

**Comment by jvhoven** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:+1: Jest tests passed: 322/322 (0 skipped)</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e93c6c406862388f4e7a588bc6010d5eb2dc74da
</p>

**Comment by MrHus** [comment]
@jvhoven ready for review.

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
   "devDependencies": {
     "@42.nl/spring-connect": "4.0.0",
     "@42.nl/react-error-store": "1.0.1",
-    "@42.nl/jarb-final-form": "1.0.0",
+    "@42.nl/jarb-final-form": "1.1.0-beta.0",
```
Comment (jvhoven) [inline package.json]: Perhaps include this in `peerDependencies` as well? Unless it is only for testing purposes of course.

Comment (MrHus) [reply package.json]: Will do, first `jarb-final-form` itself needs to be published, then I can remove the beta dependency and also change the peer dependency.

**Comment by jvhoven** [comment]
Reviewed, however I can't approve, as it is my own pull request.

**Pull request merged**